### DISCUSSION TITLE
Searching across multiple different mkdocs repositories.

### DISCUSSION THREAD

**Discussion by kirannhegde** [mkdocs/mkdocs#2766] [Q&A]
Hello there,

We have several repositories, each of which has its own docs and mkdocs.yml file. Up until now, these repositories were built independently and the generated documentation from these individual repositories was linked on a main landing page. The main landing page comes from another repository.
The problem with this approach is that, if we are on the main landing page and attempt a search in the documentation of the individual repositories, the search is not successful.  This is because, each of these different repositories have their own search index and these individual search indexes are not merged into the main search index. A user wanting to search has to navigate to these different repository from the main landing page and then use the search option to search for the relevant content.

I have attempted using the following plugin:
https://github.com/jdoiro3/mkdocs-multirepo-plugin

However, i run into the error described at: https://github.com/mkdocs/mkdocs/issues/265#issuecomment-1031630899

Are there other plugins that can help me with my problem?

I have attempted to use the mkdocs-monorepo-plugin.  The  Git repository was referenced as follows:
`'!include <url to mkdocs.yml in ther other repository>'`
However, it looks like the url mentioned here is appended to the **src **  directory and hence an error is returned stating that the path cannot be found(In this case, the error stated that the path src/<url> was not found)

Or is there an alternative approach to this issue?

My requirement is that, i would like to be able to perform searches in the docs of the other repositories from the main landing page.

Any advice would be very much appreciated. 

Thanks,
Kiran Hegde

**Comment by markallasread** [comment]
Presuming resolved given it's age, but to clarify (re: the linked comment): The mkdocs-multirepo-plugin will import the src repos, it builds after grabbing it all.
